Apps/Playground/UWP/App.cpp

```diff
@@ -215,24 +215,36 @@ void App::OnPointerMoved(CoreWindow^, PointerEventArgs^ args)
     {
         const auto& point = args->CurrentPoint;
         const auto& position = point->RawPosition;
+        const auto& deviceType = args->CurrentPoint->PointerDevice->PointerDeviceType;
+        const auto& deviceSlot = args->CurrentPoint->PointerId;
 
-        m_nativeInput->MouseMove(static_cast<int>(position.X), static_cast<int>(position.Y));
-
-        /*
-         * Note: Because PointerPressed only first for the first button press (and no additional ones)
-         * and PointerReleased only fires when all buttons have been released, we need an alternative way
-         * to track additional button presses.
-         * (See Definition - PointerPressed: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerpressed?view=winrt-22000)
-         * (See Remarks - PointerReleased: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerreleased?view=winrt-22000)
-         *
-         * In both of the below functions, we have three booleans that track each button's state and then we
-         * run these two functions to update any buttons that we may have missed.
-         */
-
-        if (point->IsInContact)
+        if (deviceType == Windows::Devices::Input::PointerDeviceType::Mouse)
         {
-            OnPointerPressed(nullptr, args);
-            OnPointerReleased(nullptr, args);
+            m_nativeInput->MouseMove(static_cast<int>(position.X), static_cast<int>(position.Y));
+
+            /*
+             * Note: Because PointerPressed only first for the first button press (and no additional ones)
+             * and PointerReleased only fires when all buttons have been released, we need an alternative way
+             * to track additional button presses.
+             * (See Definition - PointerPressed: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerpressed?view=winrt-22000)
+             * (See Remarks - PointerReleased: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerreleased?view=winrt-22000)
+             *
+             * In both of the below functions, we have three booleans that track each button's state and then we
+             * run these two functions to update any buttons that we may have missed.
+             */
+
+            // Check buttons for any button changes
+            const auto& properties = args->CurrentPoint->Properties;
+
+            if ((m_leftPressed == properties->IsLeftButtonPressed) || (m_middlePressed == properties->IsMiddleButtonPressed) || (m_rightPressed && properties->IsRightButtonPressed))
```

Comment:
Was the previous check (`IsInContact`) just wrong? Like was it missing mouse up state changes when one mouse button remains down, or something like that?

Reply:
IsInContact just defines if at least one button is pressed.  Because this check is now checking for changes to button states, we no longer needed to check IsInContact

Reply:
Reverted to IsInContact, discussed offline